File: General/break_a_palindrome.py
class Solution:
    def breakPalindrome(self, palindrome: str) -> str:
        arr = palindrome
        if(len(arr)==1):
            return ""
        i = len(arr)-2
        for i in range(len(arr)//2):
            if(arr[i]!='a'):
                return str(arr[:i]+'a'+arr[i+1:])
        return arr[:-1]+'b'

File: General/break_a_palindrome.cpp
class Solution {
public:
    string breakPalindrome(string palindrome) {
        int flag = 0;
        if(palindrome.length()<2){
            return "";
        }
        int i=0;
        while(i<palindrome.length()/2){
            if(palindrome[i]!='a'){
                palindrome[i]='a';
                flag = 1;
                break;
            }
            i++;
        }
        if(flag==0){
            palindrome[(palindrome.length())-1] = 'b';
        }
        return palindrome;
    }
};